CHANGELOG.md

```diff
@@ -1,5 +1,114 @@
 # Version changelog
 
+## 0.18.0
+
+* Derive config file name from attribute name ([#216](https://github.com/databricks/databricks-sdk-java/pull/216)).
```

Comment:
Highlight for renaming, from the PR: https://github.com/databricks/databricks-sdk-java/pull/216
```
This also renames

getAzureMSI -> getAzureMsi
setAzureMSI -> setAzureMsi
`
``